.github/ISSUE_TEMPLATE/work.yaml

```
@@ -0,0 +1,50 @@
+---
+name: Work
+description: Track any work to be done
+labels:
+  - work
+projects:
+  - UCL-MIRSG/3
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Add any new work item - this may be a feature, story, requirement,
+        service request or general task.
+  - type: textarea
+    id: short_description
+    attributes:
+      label: Short Description
+      description: Briefly summarise the work to be done
+      placeholder: What needs to be done?
+    validations:
+      required: true
+  - type: textarea
+    id: acceptance_criteria
+    attributes:
+      label: Acceptance Criteria
+      description: |
+        What must be done for this work to be considered complete?
+        These can be
+      placeholder: "- [ ] e.g. x is done in y way so that z can be achieved"
+    validations:
+      required: false
+  - type: textarea
+    id: dependencies
+    attributes:
+      label: Dependencies
+      description: |
+        What other work does this depend on? Enter the github issue
+        number in the form `- [ ] #356` or `- [ ] repo-name#243` to make
+        a checklist
```

Won't work without the org name
```suggestion
        number in the form `- [ ] #356` or `- [ ] UCL-MIRSG/repo-name#243` 
		to make a checklist
```

---

Thanks, good catch!